pom.xml

```diff
@@ -62,34 +61,34 @@
       <version>${workflow.version}</version>
       <scope>test</scope>
     </dependency>
+
     <!-- Required to avoid cyclic dependency -->
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>script-security</artifactId>
-      <version>1.13</version>
+      <version>1.29</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>matrix-project</artifactId>
-      <version>1.4.1</version>
+      <version>1.12</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>matrix-auth</artifactId>
-      <version>1.1</version>
+      <version>1.6</version>
```

Comment:
Why does 1.7 fail in tests?

Reply:
Here is the stacktrace, I dunno exactly the reason.

=== Starting testCliFailureEvenByJobAdmin(org.jenkinsci.plugins.authorizeproject.strategy.SpecificUsersAuthorizationStrategyTest)
   0.066 [id=13]	INFO	o.jvnet.hudson.test.WarExploder#explode: Exploding C:\Users\Wadeck\.m2\repository\org\jenkins-ci\main\jenkins-war\1.625\jenkins-war-1.625.war into C:\Users\Wadeck\Workspace\authorize-project-plugin\target\jenkins-for-test
  24.119 [id=13]	INFO	o.jvnet.hudson.test.JenkinsRule#createWebServer: Running on http://localhost:54205/jenkins/
  24.495 [id=20]	INFO	jenkins.InitReactorRunner$1#onAttained: Started initialization
  69.003 [id=20]	INFO	jenkins.InitReactorRunner$1#onAttained: Listed all plugins
  69.064 [id=19]	INFO	jenkins.InitReactorRunner$1#onAttained: Prepared all plugins
  69.098 [id=22]	INFO	jenkins.InitReactorRunner$1#onAttained: Started all plugins
  69.106 [id=25]	INFO	jenkins.InitReactorRunner$1#onAttained: Augmented all extensions
  73.069 [id=25]	INFO	jenkins.InitReactorRunner$1#onAttained: Loaded all jobs
  76.697 [id=20]	INFO	o.j.main.modules.sshd.SSHD#start: Started SSHD at port 54230
  76.697 [id=20]	INFO	jenkins.InitReactorRunner$1#onAttained: Completed initialization
  82.934 [id=15]	WARNING	hudson.Util#warnWindowsSymlink: Symbolic links enabled on this platform but disabled for this user; run as administrator or use Local Security Policy > Security Settings > Local Policies > User Rights Assignment > Create symbolic links
  83.049 [id=71]	INFO	h.TcpSlaveAgentListener$ConnectionHandler#run: Accepted connection #1 from /127.0.0.1:54246
WARN: The method class org.apache.commons.logging.impl.SLF4JLogFactory#release() was invoked.
WARN: Please see http://www.slf4j.org/codes.html#release for an explanation.

```
java.lang.AssertionError: hudson.security.AccessDeniedException2: test1 is missing the Job/ExtendedRead permission
	at hudson.security.ACL.checkPermission(ACL.java:63)
	at hudson.model.AbstractItem.checkPermission(AbstractItem.java:498)
	at hudson.cli.GetJobCommand.run(GetJobCommand.java:46)
	at hudson.cli.CLICommand.main(CLICommand.java:237)
	at hudson.cli.CliManagerImpl.main(CliManagerImpl.java:92)
	...
```
 
Expected :0
Actual   :-1
 <Click to see difference>

```
	at org.junit.Assert.fail(Assert.java:88)
	...
```

Reply:
EXTENDED_READ permission needs to be explicitly enabled